src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -75,49 +78,65 @@ export const RegistrationFormActions = ({
         }}>
         {tabNumber > RegistrationTab.Description && (
           <Box sx={{ gridArea: 'back-button' }}>
-            <Button
-              variant="outlined"
-              data-testid="button-previous-tab"
-              startIcon={<ArrowBackIcon />}
-              onClick={() => setTabNumber(tabNumber - 1)}>
-              {tabNumber === RegistrationTab.ResourceType && t('registration.heading.description')}
-              {tabNumber === RegistrationTab.Contributors && t('registration.heading.resource_type')}
-              {tabNumber === RegistrationTab.FilesAndLicenses && t('registration.heading.contributors')}
-            </Button>
+            <Tooltip title={t('common.previous')}>
+              <IconButton onClick={() => setTabNumber(tabNumber - 1)} disabled={isFirstTab}>
+                <Box
+                  sx={{
+                    bgcolor: isFirstTab ? 'gray' : 'info.main',
+                    display: 'flex',
+                    borderRadius: '50%',
+                    padding: '0.2rem',
+                  }}>
+                  <ChevronLeftIcon sx={{ color: 'white' }} />
+                </Box>
+              </IconButton>
+            </Tooltip>
           </Box>
         )}
+
         <Button data-testid="open-support-button" onClick={toggleSupportModal} sx={{ gridArea: 'support-button' }}>
           {t('common.support')}
         </Button>
         {tabNumber < RegistrationTab.FilesAndLicenses ? (
```

Comment:
Siden vi nå har en variabel som i praksis er det samme
```suggestion
        {!isLastTab ? (
```